src/utils/payments/pawapay/pawapay.ts

```
@@ -0,0 +1,473 @@
+import crypto from 'crypto';
+
+
+// PawaPay API client with proper URL and signature support
+class PawaPay {
```

ESLint

[error] 'PawaPay' is defined but never used. Allowed unused vars must match /^_/u.

(unused-imports/no-unused-vars)